code/modules/atmospherics/machinery/components/fusion/hfr_main_processes.dm

```diff
@@ -548,8 +548,10 @@
 
 	if(moderator_list[/datum/gas/oxygen] > 150)
 		if(iron_content > 0)
-			iron_content = max(iron_content - 0.5, 0)
-			moderator_internal.gases[/datum/gas/oxygen] -= 10
+			var/max_iron_removable = 0.00415
```

Comment:
make this a define

Reply:
Done, killed a few more magic numbers in the area too